lib/libthr/thread/thr_create.c

```diff
@@ -58,7 +58,13 @@ __FBSDID("$FreeBSD$");
 #include "thr_private.h"
 
 static int  create_stack(struct pthread_attr *pattr);
+#if defined(__CHERI_PURE_CAPABILITY__) && defined(RTLD_SANDBOX)
+void thread_start(struct pthread *curthread);
```

Comment:
I'm trying to solve this by keeping `thread_start` as a static function and exporting `_thread_start` by doing
```
__attribute__((weak,alias("thread_start"))) void _thread_start(struct pthread *curthread);
```
which can be referenced in `thr_rtld.c`.

Strangly, using `__weak_reference(thread_start, _thread_start);` instead of the attribute will cause the warning `unused function thread_start` despite the two being equivalent (I think?),